### DISCUSSION TITLE
New TURBO RANKED MODE!

### DISCUSSION THREAD

**Discussion by Vittor-Javidan** [ValveSoftware/Dota2-Gameplay#21650] [Ideas]
This is an old issue started by @andrestrindade, suggesting a turbo ranked mode. A lot of players support this idea!

Bellow the link to the issue with where the discussion occurred, and the quoted text created by @andrestrindade himself:

https://github.com/ValveSoftware/Dota2-Gameplay/issues/8751

> I am writing to suggest a new addition to the game that I believe would greatly enhance the player experience: a Ranked Turbo Mode.
> 
> Turbo Mode has become increasingly popular among Dota 2 players as a way to play faster, more action-packed games. However, the lack of ranking system in Turbo Mode makes it less appealing to competitive players who enjoy the thrill of climbing the ranks.
> 
> By introducing a Ranked Turbo Mode, players can enjoy the fast-paced gameplay of Turbo Mode while also being able to compete for higher ranks and rewards. This would attract a new wave of competitive players who may have previously overlooked Turbo Mode.
> 
> Additionally, the shorter game time in Turbo Mode makes it more accessible to players who may not have hours to spend on each game. The Ranked Turbo Mode would be perfect for those players who want to enjoy a quick and intense game while also having a chance to climb the ranks.
> 
> I believe that the addition of a Ranked Turbo Mode would greatly enhance the Dota 2 experience for both casual and competitive players alike. It would provide a new and exciting way to play the game, and attract a wider range of players to the community.
> 
> Thank you for considering this suggestion, and I look forward to seeing what the future holds for Dota 2.

**Comment by andrestrindade** [comment]
I am still waiting

**Comment by lokize** [comment]
Turbo mode is the standard in the future; no one over 30 has much time to play matches that last an hour.

**Comment by PlayerNoon** [comment]
that would be cool. i play only turbo, other modes are too boring.

**Comment by SadicVicious** [comment]
i sent this to dota 
Message from you on Sep 7 @ 4:06pm | 4 days ago
I think the developers should create a Ranked Turbo mode, and I will prove the reasons why Ranked Turbo should exist.

First reason: Faster games - Whether you like it or not, people are migrating to other games outside the MOBA genre because they have less time. Dota demands that you stay in front of your PC until the match is over. Additionally, we see today that few games require long play sessions.

Second reason: Dota was a mod of Warcraft - When Dota 2 was released, I was one of the players who found it slow. At the same time, Heroes of Newerth (HoN) was created, and many migrated to HoN because the hero plays were faster (at the time). HoN is a great example of a MOBA with a turbo ranked mode—it had both ranked modes, each with its own medal system. So, this is not some crazy idea of mine; turbo ranked modes have existed before.

Third and personal reason: I’m an anxious person, and since I switched from classic to turbo, I’ve found it a thousand times more enjoyable and faster to complete items. I remember when I played "classic," I rarely managed to fully build my hero, often finishing with only about three items. It leaves you wanting to test the full potential of the hero.

Fourth reason (for the developers): What harm would it do? None. It would simply create a new division of players. Someone could choose to be ranked in both modes, and you could create more tournaments. It would be a fresh addition, something that hasn't happened in a while.

So, that’s it; I don’t want to go on too long. If, in the future, you find that it interferes with regular ranked, you could make it a seasonal feature, and end the season if needed.

**Comment by tgharib** [comment]
I agree. I don't have the time these days to play regular long dota games.

**Comment by Mudspiritual** [comment]
Please spam this message on dota 2 issues, hopefully they will implement it in the future
Why a Ranked Turbo Mode Could Benefit Dota 2’s Future”

Proposed Post Content:

Hello everyone,

I’d like to revive and expand on the discussion around introducing a Ranked Turbo Mode to Dota 2—an idea first brought up by other community members like @andrestrindade. Over the years, Turbo has attracted a growing player base that appreciates shorter, more action-packed matches. However, because it currently lacks a ranked ladder, many competitive-minded players overlook it. Here are a few reasons why I believe Ranked Turbo could be a welcome addition:

Faster Games for Busy Schedules

As more players find themselves short on free time, committing to a 40–60 minute standard or ranked match can be impractical.
A ranked Turbo environment would cater to those who still love Dota 2’s depth but need a condensed match format, without sacrificing the competitive spirit.
Accessibility & Wider Appeal

Players over 30 or those juggling work, studies, or family responsibilities often cite time constraints as a barrier to playing traditional Dota.
By offering a formal ranked system in Turbo, Dota 2 becomes more inviting to a broader audience—including both new players and veterans seeking a quick, high-energy experience.
Precedent in Other MOBAs

Historically, other MOBA titles (such as Heroes of Newerth) have used a similar approach, offering both classic ranked modes and faster-paced ranked modes.
This demonstrates that faster-ranked game modes can coexist with traditional ones, often increasing overall community engagement.
Retaining Competitive Depth

Contrary to fears that quicker games might lessen strategic depth, Turbo’s fast progression can actually amplify certain aspects of the game (e.g., itemization choices, early team fights).
A ranked version of Turbo would still reward skill, decision-making, and hero mastery; it would merely place these elements in a more rapid context.
Low Risk to Standard Modes

If there is a concern about splitting the player base, this mode could be introduced as a seasonal or experimental feature first, similar to how other limited-time events and modes have been handled in Dota 2’s history.
If the mode proves successful, it could then be integrated permanently, fostering distinct but complementary competitive experiences.
New Incentives & Tournaments

A Ranked Turbo Mode could open up new opportunities for event organizers to host tournaments or leagues exclusively for faster matches.
This would bring additional variety to the Dota 2 esports scene and highlight talented players who excel in rapid-play environments.
Ultimately, Ranked Turbo is about recognizing that Dota 2’s community is diverse—encompassing everything from hardcore players who love long, methodical games to those who crave shorter, more intense sessions. Providing a competitive Turbo ladder would allow more players to chase ranks, earn rewards, and stay engaged without the time commitment of the standard format.

Thank you for considering this suggestion. I hope the Dota 2 development team and community will continue to discuss the potential merits of Ranked Turbo, and I look forward to seeing how Dota 2 evolves in the future.

Kind regards,
A fellow Dota 2 enthusiast

**Comment by superjacques** [comment]
agree with the above

**Comment by armendimeri** [comment]
yes

**Comment by katchwi-arch** [comment]
no, remove ranked altogether because the system is broken
